### ISSUE TITLE
Zooming does not work with perfect scrollbar 

### ISSUE THREAD

**Issue by deje1011** [utatti/perfect-scrollbar#436]
First of all, thank you for this great plugin!
Unfortunately it is not possible to zoom in/out using a trackpad or ctrl+scroll while hovering over an area that uses perfect scrollbar. Is there a way to fix this?

Edit: 
QuickFix: If event.preventDefault is not called inside the mousewheelHandler function, it works as expected. 

```
....
    updateGeometry(element);

    shouldPrevent = (shouldPrevent || shouldPreventDefault(deltaX, deltaY));
    if (shouldPrevent) {
      e.stopPropagation();
      //e.preventDefault();
    }
}
```

Edit 2: I'm using version 0.6.5 by the way (and did not update yet because I had to make some other adjustments)

Edit 3: Not calling preventDefault on the wheel event reenables the "back" gesture on MacBooks, which is super annoying... 

Edit 4: I found a solution that works perfectly for my purposes so far. 

```
// If deltaX is 0, the user probably wants to zoom in which case we should not call preventDefault
shouldPrevent = (shouldPrevent || shouldPreventDefault(deltaX, deltaY)) && (Math.abs(deltaX) > 0);
if (shouldPrevent) {
  e.stopPropagation();
  e.preventDefault(); 
}
```

Maybe there should be an option to allow the back gesture if element.scrollLeft is 0?

**Comment by noraesae** [comment]
I'm really sorry for the late response.

This is very true as it cannot tell if a wheel event is triggered by just a normal scrolling or a pinch. I'm not sure if there is a JS way to tell them. Any idea?

**Comment by deje1011** [comment]
The only fix I came up with so far is the "delta-check" (Edit 4). But suddenly it does not work anymore (after I updated to 0.6.10 I guess). Also, I can't get the gulp tasks to work properly anymore. They seem to be working but my changes aren't reflected in the output... If I find the time, I will look into it a little more next week.

**Comment by iconexperience** [comment]
While detecting a pinch gesture may be difficult, it should be easy to detect if the user holds the `ctrl` key during scrolling uing `MouseEvent.ctrlKey`, no? I am a Windows user, but `ctrl`+mouse wheel is used for zooming on Mac OSX, too, I think. This would at least cover zooming for mouse/keyboard usage.

**Comment by utatti** [comment]
Fixed by #709
